src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -29,6 +33,18 @@
         return roomsSeederList;
     }
 
+    private static List<Reservation> newReservationsList(int index) {
+        List<Reservation> reservationList = new ArrayList<>();
+        for (int j = 0; j < 12; j++) {
+            reservationList.add(new Reservation(new Date(117, j, 10),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "Date"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)